src/schema/v2/artwork/__tests__/artwork.test.js

```diff
@@ -2437,11 +2437,24 @@ describe("Artwork type", () => {
       })
     })
 
-    it("returns false when artwork requires vat and partner is not vat registered", () => {
+    it("returns false when artwork requires VAT and partner does not have a VAT status", () => {
       artwork.vat_required = true
       artwork.partner = { id: "123" }
       context.partnerAllLoader = jest.fn(() =>
-        Promise.resolve({ vat_registered: false })
+        Promise.resolve({ vat_status: null })
```

Comment:
so `vatRequirementComplete` now requires that a vat status be _present_, but doesn't make demands beyond that? If so I wonder if we could add another test for the `true` case in addition to this one:
https://github.com/artsy/metaphysics/pull/3568/files#diff-1bca24dc978ef5f6e034b1bf83eb9f55a14bcb7181e8bc18df4e6790ed5e520dR2416-R2425

Reply:
> so vatRequirementComplete now requires that a vat status be present, but doesn't make demands beyond that?

Yes, we only require some status to be set for the new relaxed requirements.

Is [this test](https://github.com/artsy/metaphysics/pull/3568/files#diff-1bca24dc978ef5f6e034b1bf83eb9f55a14bcb7181e8bc18df4e6790ed5e520dR2427) you're referring to, or you think we can add more coverage?
```javascript
it("returns true when artwork requires VAT and partner has any VAT status")
```